grapher/Models/Calculations/AccelChartData.cs

```
@@ -80,6 +81,8 @@ public void Clear()
 
         public int GetVelocityIndex(double outVelocityValue)
         {
+            Debug.Assert(outVelocityValue >= 0);
```

Better to throw an exception with a good message than to use an assertion